js/notify.js

```diff
@@ -15,7 +15,7 @@ define([
             level: type == 'error' ? 'danger' : type,
             message: message
         }));
-        Log.debug('Notify:', {type: type, message: message, container: container});
+        log.debug('Notify:', {type: type, message: message, container: container});
```

Comment:
Are you sure about that because `Log` is the second parameter of the function and equals to `'logger'`?